generate/gen_ext2

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh -ef
 
-SIZE_BLOCKS=$((64 * 1024))
-DAT="$DATA_DIR/ext2.dat"
+cd "$DATA_DIR"
```

Comment:
Any reason on why are you inlining the constants?

Reply:
More readable.